gtnet/sequence.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import skbio
+
+__all__ = ['_get_DNA_map',
+           'get_sequences',
+           'chunk_seq',
+           'get_rev_seq',
+           'get_bidir_seq']
+
+
+def _get_DNA_map():
```

Comment:
Add argument to this function so that it can take a vocabulary string as an argument (you can default it to what is currently used). Compute the basemap, and then compute the revcomp map, and return it. You will need to use code from.

https://github.com/exabiome/deep-taxon/blob/enh/parallel_inf/src/exabiome/sequence/dna_table.py#L600-L633

Reply:
included the `get_revcomp_map` and now a different vocabulary can be used. If none is passed, default is what was previously hardcoded